staffjoy/resources/location.py

```diff
@@ -25,3 +28,12 @@ def get_manager(self, id):
     def create_manager(self, **kwargs):
         """Typically just pass email"""
         return Manager.create(parent=self, **kwargs)
+
```

Comment:
Can you comment in that these are read-only endpoints intended for visualization?